src/features/classification/security-controls/SecurityControlsModal.js

```diff
@@ -38,6 +39,12 @@ const SecurityControlsModal = ({
     }
 
     const title = <FormattedMessage {...messages.modalTitle} values={{ itemName }} />;
+    const securityControlsItems = modalItems.map(({ message, tooltipMessage }) => {
+        if (message) {
+            return <SecurityControlsItem key={message.id} message={message} appNames={tooltipMessage} />;
+        }
+        return null;
```

Comment:
Same here, this can probably be avoided by having `message` property as required and not returning an item at all when there's no message